### PR TITLE
Fix: Typo for docs, client, output_manager

### PR DESCRIPTION
**Pull request by Digitalisx**
Hello, everyone in the community! 🙂
I found some typos during code analysis and fixed them.
I hope it's a good first opportunity to contribute to turbinia.

### CODE REVIEW

File: turbinia/client.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def create_recipe(
     If no recipe_name is specified, this  method returns a default recipe.
     If a recipe_name is specified then this method will build the recipe
     dictionary by reading the  contents of a recipe file. The path to
-    the recipe file is inferre from the recipe_name and the RECIPE_FILE_DIR
+    the recipe file is inferrer from the recipe_name and the RECIPE_FILE_DIR
```
Comment (aarontp) [inline turbinia/client.py:178]: I think this should be "inferred"

**Review by aarontp — COMMENTED** [review]
Hi @Digitalisx , thanks for the contribution!   I left one comment about one of the changes.

Could you also please take a look at the CLA link (https://github.com/google/turbinia/pull/1029/checks?check_run_id=5806647056) that was added by the `google-cla` bot on this PR?  Thanks!

**Comment by Digitalisx** [comment]
@aarontp Thank you for your review, I reflected the correction you requested.

**Comment by aarontp** [comment]
Thanks @Digitalisx .  I think you may need to go to https://cla.developers.google.com/ and sign the CLA agreement there in order for us to accept/merge this code (you should only need to do this once).  Thanks!

**Comment by Digitalisx** [comment]
@aarontp It seems that I have already entered and agreed to the personal information in Google CLA. Please let me know if you need additional action!

**Review by aarontp — APPROVED** [review]
Ok, I see that it's showing up as signed now.  Thanks much!  I hope we see more contributions from you in the future as well :).   FYI, many of us are in the "Open source DFIR" slack Turbinia channel if you have any questions, etc.

**Pull request merged**